ui/src/core/xibo-cms.js

```diff
@@ -3079,6 +3087,103 @@ function createColorPicker(element, options) {
   }, options));
 }
 
+
+/**
+ * Create custom date range filter
+ * @param {object} element jquery object or CSS selector
+ */
+function updateDateRangeFilter(element) {
+  const selected = $(element).find('option:selected').val();
+  const $form = $(element).closest('form');
+
+  // This would allow us to target this specific element in case there are multiple range filters used
```

Comment:
All looks good in this function, it's just this line that has more than 80 char of lenght (https://eslint.org/docs/latest/rules/max-len). Can you please split it into 2 comments?